internal/controller/streams.go

```diff
@@ -1007,6 +1014,15 @@ func ConnectStreams(ctx context.Context, logger *zap.Logger, envMap EnvironmentC
 			go func() {
 				defer close(flowCacheOutReaderDone)
 
+				// wait until the flow collector is initialized, or bail if the context is canceled
+				select {
+				case <-sm.networkFlowsReady:
+					// proceed
+				case <-authConContext.Done():
+					logger.Info("Failed to start sending network flows from cache", zap.Error(authConContext.Err()))
+					return
```

Comment:
Make the linter happy:
```suggestion

					return
```